app1.js

```diff
@@ -54,7 +55,12 @@ function lightsaberColor(){
 
 function writeImageToDocument(imgUrl, imgAlt) {
     let tag = `<img src="{imgUrl}" alt="{imgAlt}" />`;
-    document.write(tag);
+
+    writeContentToDocument(tag);
+}
+
+function writeContentToDocument(content) {
```

Comment:
So now we have a method to write just any content...it doesn't create tags or attributes, just straight up writes...we can combine that with the last change to create a 'chain' of methods that drill down to the work being down. This isn't an uncommon thing to see: Various methods that do 'heavy lifting' and setup that wrap around simpler functions that aggregate all the work.